### PR TITLE
Admin logic gk

### PR DESCRIPTION
**Pull request by gknisely**
@dgearhart @dnesbitt61 @kevinkreiser 

Init logic for admins.  Next, need to save to sqlite

### CODE REVIEW

File: valhalla/mjolnir/osmadmin.h [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+  // Count of members.
+  uint16_t memberid_count_;
+
+  // Name of country or state/prov
+  std::string name_;
+
+  // Admin level.  2 = country; 4 = state.
+  uint8_t admin_level_;
+};
```
Comment (dnesbitt61) [inline valhalla/mjolnir/osmadmin.h]: Might get smaller size/better data packing if move admin_level above string

**Comment by dnesbitt61** [comment]
Looks fine. Do we get all the admin information we need from OSM or are there different/better sources?

File: valhalla/mjolnir/osmadmin.h (hunk 1 above)
Comment (gknisely) [inline valhalla/mjolnir/osmadmin.h]: Ok will do.

**Comment by gknisely** [comment]
Should have all the info we need minus timezone info.

**Comment by dgearhart** [comment]
:+1:

**Pull request merged**